app/containers/Navigation.android.js

```diff
@@ -1,10 +1,11 @@
 'use strict'
 
 import React, { Component } from 'react';
-import { View } from 'react-native'
+import { View, Alert } from 'react-native'
 
 import { connect } from 'react-redux';
 import autobind from 'autobind-decorator';
+import FCM from 'react-native-fcm';
```

Comment:
This needs to be changed to
``import * as FCM from 'react-native-fcm';``

Otherwise the app won't start as it tries to import a function named FCM, not the whole module.